nautobot_netbox_importer/tests/test_import_objectchange.py

```diff
@@ -0,0 +1,109 @@
+"""Test the importing functionality of nautobot-netbox-importer."""
+
+import os
+import json
+import copy
+from unittest.mock import Mock
+
+import yaml
+from django.core.management import call_command
+from django.contrib.contenttypes.models import ContentType
+from django.test import TestCase
+from nautobot.extras.models import ObjectChange
+from nautobot.extras import models
+from nautobot_netbox_importer.management.commands.import_netbox_objectchange_json import Command
+from .test_import import TestImport
+
+
+NETBOX_DATA_FILE = os.path.join(os.path.dirname(__file__), "fixtures", "netbox_dump.json")
+NETBOX_OBJECTCHANGE_DATA_FILE = os.path.join(os.path.dirname(__file__), "fixtures", "netbox_objectchange_dump.json")
+NAUTOBOT_DATA_FILE = os.path.join(os.path.dirname(__file__), "fixtures", "nautobot_objectchange_expectations.yaml")
```

Comment:
To avoid the need to duplicate the (non-ObjectChange) expectations across two YAML files, consider either:

1) Only including the ObjectChange records in `nautobot_objectchange_expectations.yaml` (since the other records are already being tested by the other test case and shouldn't be any different here), or
2) Maintain two separate YAML files and just load them both together, i.e.:
    ```python
    with open(NAUTOBOT_DATA_FILE, "r") as handle:
        cls.nautobot_data = yaml.safe_load(handle)
    with open(NAUTOBOT_OBJECTCHANGE_DATA_FILE, "r") as handle:
        cls.nautobot_data += yaml.safe_load(handle)
    ```

Reply:
Ok I will go for the second option, just to add the extra verification that the new command doesn't break anything from the first